modules/apps/object/object-test/src/testFunctional/paths/ObjectRelationships.path

```diff
@@ -20,6 +20,11 @@
 		<td>//button[@class = 'dropdown-item']//div[contains(@class, 'd-flex')][contains(text(),'${key_parameterOption}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>SELECT_PARAMETER_OPTION_IN_RELATIONSHIP</td>
+		<td>//div[@id='clay-dropdown-menu-3']//button[@class = 'dropdown-item']//div[contains(text(),'${key_parameterOption}')]</td>
+		<td></td>
+	</tr>
```

Comment:
This path is not necessary for this step. You can use the same path on the macro. See: `"ObjectRelationships#SELECT_PARAMETER_OPTION"`. Try it and see if works.